internal/test/must.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+)
+
+// test.T(t).Digest(img)
+//
+// or
+//
+// must := must.T(t)
+// must.Digest(img)
+
+// func TestFoo(t *testing.T) {
+//   must := test.T(t)
+//   img := must.RandomImage(1024, 5)
```

Comment:
```suggestion
//   img := must.Image(random.Image(1024, 5))
```

Reply:
Dropped this comment section anyway, in favor of the test, which will actually stay up to date.